sdk/storage/azure-storage-file-datalake/tests/test_file_async.py

```diff
@@ -562,16 +653,20 @@ async def test_read_file_with_user_delegation_key_async(self, datalake_storage_a
                                       expiry=datetime.utcnow() + timedelta(hours=1),
                                       )
 
-        # doanload the data and make sure it is the same as uploaded data
+        # download the data and make sure it is the same as uploaded data
         new_file_client = DataLakeFileClient(self.account_url(datalake_storage_account_name, 'dfs'),
                                              file_client.file_system_name,
                                              file_client.path_name,
                                              credential=sas_token)
         downloaded_data = await (await new_file_client.download_file()).readall()
-        self.assertEqual(data, downloaded_data)
+        assert data == downloaded_data
 
     @DataLakePreparer()
-    async def test_read_file_into_file_async(self, datalake_storage_account_name, datalake_storage_account_key):
+    @recorded_by_proxy_async
+    async def test_read_file_into_file(self, **kwargs):
```

Comment:
Like sync, can remove concurrency from this test and the next one. They are already recorded which is good.